test/functional/sqlsrv/MsHelper.inc

```diff
@@ -382,6 +382,36 @@ function createTable($conn, $tbname, $columnMetaArr)
 }
 
 /**
+ * Create a table with options for sqlsrv_prepare()
+ * @param  resource  $conn : sqlsrv connection resource
+ * @param  string  $tbname : name of the table to be created
+ * @param  array  $columnMetaArr : array of ColumnMeta objects, which contain metadata for one column
+ * @param  array  $options : array of options for sqlsrv_prepare()
+ * @return resource sqlsrv statement resource
+ */
+function createTableEx($conn, $tbname, $columnMetaArr, $options)
```

Comment:
What does createTable do then?

Reply:
I added this extra method that takes options plus using sqlsrv_prepare/execute()
But with hindsight I think it's really unnecessary (at least for the test sqlsrv_buffered.phpt)